### PR TITLE
Add openfe-analysis

### PR DESCRIPTION
**Pull request by mikemhenry**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by mikemhenry** [comment]
You guys know the drill:
@dwhswenson
@ialibay
@richardjgowers

**Comment by richardjgowers** [comment]
Yes I am happy to maintain this recipe

**Comment by IAlibay** [comment]
Yes I am happy to maintain this feedstock.

File: recipes/openfe-analysis/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  requires:
+    - pip
+  commands:
+    - pip check
+
+about:
+  home: https://github.com/OpenFreeEnergy/openfe_analysis
+  summary: 'Analysis tools for the openfe ecosystem'
+  description: |
+    simplejson is a simple, fast, complete, correct and extensibledd
```
Comment (richardjgowers) [inline recipes/openfe-analysis/meta.yaml:40]: @mikemhenry oi

Comment (mikemhenry) [reply recipes/openfe-analysis/meta.yaml:40]: goodness me I thought this was going to be the one I get right in a commit

**Comment by dwhswenson** [comment]
Yep, happy to be a maintainer.

**Comment by mikemhenry** [comment]
@conda-forge/help-python ready for review!

**Comment by mikemhenry** [comment]
@conda-forge/help-python ready for review!

File: recipes/openfe-analysis/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/openfreeenergy/openfe_analysis/archive/{{ version }}.tar.gz
+  sha256: 74afe903ef9e9095a59b48ee5e06a6f903d10cd693cb19ca2f0f3d7c8f2fe80e
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [py==38]
```
[the remark this answers is not in the document]
Comment (mikemhenry) [reply recipes/openfe-analysis/meta.yaml]: It should be `noarch`. I am not sure how I made that mistake. Just fixed that, sorry about that!

**Comment by mikemhenry** [comment]
@conda-forge/help-python , ready for review!

Thanks! @ocefpaf

**Comment by mikemhenry** [comment]
Windows isn't important/expected. The error is `  [ERROR   ] versioningit: NotVCSError: C:\bld\openfe-analysis_1696284850125\work is not in a Git repository` I am not sure why the fallback method isn't working on windows like it is on osx and linux, but we don't need this package on windows

**Pull request merged**